addons/website_sale/views/templates.xml

```diff
@@ -175,7 +175,7 @@
                     <t t-set="bg_color" t-value="td_product['ribbon']['bg_color'] or ''"/>
                     <t t-set="text_color" t-value="td_product['ribbon']['text_color']"/>
                     <t t-set="bg_class" t-value="td_product['ribbon']['html_class']"/>
-                    <span t-attf-class="o_ribbon #{bg_class}" t-attf-style="#{text_color and ('color: %s; ' % text_color)}#{bg_color and 'background-color:' + bg_color}" t-field="product.website_ribbon_id.html"/>
+                    <span t-attf-class="o_ribbon o_not_editable #{bg_class}" t-attf-style="#{text_color and ('color: %s; ' % text_color)}#{bg_color and 'background-color:' + bg_color}" t-field="product.website_ribbon_id.html"/>
```

Comment:
```suggestion
                    <span t-attf-class="o_ribbon #{bg_class}" t-attf-style="#{text_color and ('color: %s; ' % text_color)}#{bg_color and 'background-color:' + bg_color}" t-esc="product.website_ribbon_id.html"/>
```
Is there a point to keep using the `t-field` if it's not going to be editable anyway?

Reply:
The purpose of `t-field` was to make the ribbon savable which is important to mark it as dirty and save the ribbon of a product (see https://github.com/odoo/odoo/pull/135008)

Reply:
Alright then the pr is LGTM (a test would be welcome at some point tho)